rpcserver.go

```diff
@@ -4521,14 +4533,20 @@ func (r *rpcServer) DescribeGraph(ctx context.Context,
 			nodeAddrs = append(nodeAddrs, nodeAddr)
 		}
 
-		resp.Nodes = append(resp.Nodes, &lnrpc.LightningNode{
+		lnNode := &lnrpc.LightningNode{
 			LastUpdate: uint32(node.LastUpdate.Unix()),
 			PubKey:     hex.EncodeToString(node.PubKeyBytes[:]),
 			Addresses:  nodeAddrs,
 			Alias:      node.Alias,
 			Color:      routing.EncodeHexColor(node.Color),
 			Features:   invoicesrpc.CreateRPCFeatures(node.Features),
-		})
+		}
+
+		if req.CalculateNodeCentrality {
+			lnNode.BetweennessCentrality = centrality[node.PubKeyBytes]
```

Comment:
When running this, I find that some nodes have a value of zero. Is it the case that these nodes are just unconnected, or that they don't have at least _two_ channels?

Reply:
Also I'm getting rather large values as well, like `2738.2895441051587` as an example. I was under the impression that centrality would output a value between 0 and 1, as it's itself a ratio of the total number of paths relative to the total number that pass through a target node.

Reply:
How about we normalize things so we have a value between 0 and 1 which'll make the value easier to interpret (most connected node has the highest relative value, my node has a smaller value, but not my orders of magnitude). The wiki page for betweenness centrality has a suggested normalization algorithm.

Reply:
re: what does a centrality value of zero mean?

This means that no shortest paths pass trough that node. Note that as per the definition, (for reference: sum(sigma(s,t) / sigma(s,t,v)) [s != t != v]) leaf nodes don't have shortest paths passing trough, therefore their centrality is always zero.

Reply:
re: why aren't values between 0 and 1?

The definition is a bit tricky, but if you take a closer look at it in reality betweenness centrality isn't a ratio, but a sum of ratios for all distinct pairs of vertices. We may normalize the centrality values, eg. by using the factor 2/(n*(n-1)), or with a better normalizing factor if needed but it may get harder to interpret for humans.

Reply:
Gotcha, IMO normalization is desirable, as otherwise it's hard to compare two values between distinct nodes. As an example, one of my personal nodes has a value of 26k, but without scanning all the other values for a max, I have no idea where I stand compared to all the other nodes.

Reply:
I agree, added min/max normalization. To be fair though, one still has to scan other nodes to figure out ranking.